Makefile

```diff
@@ -29,7 +29,8 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # ibm.com/rsct-operator-bundle:$VERSION and ibm.com/rsct-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= ibm.com/rsct-operator
+#IMAGE_TAG_BASE ?= ibm.com/rsct-operator
```

Comment:
remove comment, how about pushing this image to `quay.io/powercloud/rsct-operator`?

Reply:
sure, I will update it